Cargo.toml

```diff
@@ -26,42 +26,61 @@ homepage = "https://xilem.dev/"
 # Remove assigning_clones once it's allowed by default in stable Rust
 # https://github.com/rust-lang/rust-clippy/pull/12779
 clippy.assigning_clones = "allow"
+# Remove assigning_clones once it's allowed by default in stable Rust
+# https://github.com/rust-lang/rust-clippy/pull/12779
```

Comment:
Why all the comments (including below)?

Reply:
Oh seriously? I trusted `cargo upgrade` to actually do the right thing. That's incredibly annoying.

Reply:
Ok, that's a weird, probably open an issue in cargo upstream then?

Reply:
It's part of [cargo-edit](https://github.com/killercup/cargo-edit). I believe that the in-place toml editing doing stupid things with comments/ordering is known, although I can't find where it's tracked at the moment.